Formula/wskdeploy.rb

```diff
@@ -0,0 +1,24 @@
+class Wskdeploy < Formula
+  desc "Apache OpenWhisk project deployment utility"
+  homepage "http://openwhisk.org/"
+  url "https://github.com/apache/incubator-openwhisk-wskdeploy/archive/0.9.7.tar.gz"
+  sha256 "15f586aeb3221e67b583941b988fcc572eebafc64c58d4a2df2787007344f064"
+
+  depends_on "go" => :build
+  depends_on "godep" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath / "src/github.com/apache/incubator-openwhisk-wskdeploy").install buildpath.children
+    cd "src/github.com/apache/incubator-openwhisk-wskdeploy" do
+      system "godep", "restore"
+      system "go", "build", "-o", bin / "wskdeploy",
+             "-ldflags", "-X main.Version=#{version}"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/wskdeploy version")
```

Comment:
I've tried this but it seems that all commands need a live OpenWhisk server to execute against. Running one locally as part of the test seems like overkill. Embedding real tokens etc. is a bad idea. And any tests without a live server error out so I'm not quite sure what to do to be honest. Do you have any recommendations @zbeekman?

Reply:
[This](https://github.com/apache/incubator-openwhisk-wskdeploy/blob/master/docs/wskdeploy_package_minimal.md#dry-run-mode) seems like a viable option.

Reply:
@DomT4 thats what I also initially thought (it was in one of the squashed commits). But that still makes a roundtrip to the server to check to see if authentication works.

Reply:
Yes, @Juice10, a better test would really be good to have. Can you take a look at the dry-run option mentioned at the link above ☝️

Reply:
We both commented at the same time. Let's see what Dominyk says.

Reply:
@zbeekman looks like we both typed a comment at the same moment. See my response to DomT4 above.

Reply:
😸. 

> But that still makes a roundtrip to the server to check to see if authentication works.

That's fairly annoying, from a testing perspective. Can we run a server locally from within this formula or is that going to involve other formulae/remote servers/etc?

Reply:
I agree its fairly annoying from a testing perspective. Downside of OpenWhisk is that it is a bit of a beast with lots of components and moving parts (think of it as being the equivalent in size and components to something like kubernetes). 
The easiest install we could do would be with Docker Composer which installs a whole slew of docker images. We could do it, but it would bring the testing time up from a minute, to about an hour or so depending on how beefy the Jenkins machine we are running is.

Reply:
@DomT4 @zbeekman I finally figured it out, `wskdeploy` does need a live server to talk to but doesn't need to send it credentials. Meaning we don't have to run one locally and we don't have to expose real credentials.
Check out the commit I did, if you like it I'll squash the commits again.

Reply:
FYI, the new tests test way more than just the version number:
https://github.com/Homebrew/homebrew-core/blob/16eeb439da6e2cce3a4f32450de9baeb549d8df4/Formula/wskdeploy.rb#L22-L35